src/main/java/filtering_use_case/screens/ChoicesSortedView.java

```diff
@@ -109,9 +109,9 @@ public void back() {
     }
 
     @Override
-    public void home() {
+    public void home(User user) {
```

Comment:
You don't need to reimplement the home method it is already implemented this way in the parent class

Reply:
I didn't realize this, if I have the time I will go back and change this